src/utils.ts

```diff
@@ -57,3 +58,48 @@ export function callEachWith (callbacks: Function[], arg0?: any) {
     cb(arg0)
   }
 }
+
+const isBrowser = typeof window !== 'undefined'
+
+/** Start debugging hook names and timing in console */
+export function createDebugger (hooks: Hookable, _options: CreateDebuggerOptions = {}) {
+  const options = {
+    inspect: isBrowser,
+    group: isBrowser,
+    filter: () => true,
+    ..._options
+  }
+
+  const _filter = options.filter
+  const filter = typeof _filter === 'string' ? (name: string) => name.startsWith(_filter) : _filter
+
+  const wrapName = options.tag ? (event: string) => `[${options.tag}] ${event}` : (event: string) => event
+
+  const unsubscribeBefore = hooks.beforeEach(({ name }) => {
+    if (!filter(name)) { return }
+
+    console.time(wrapName(name))
```

Comment:
Refactor? Can we make it simpler by making precomputed const `logPrefix` and use console.method(logPrefix + name) and avoid function call?